### PR TITLE
POWR-3879 update theme to Bootstrap 5.1 and apply Portland.gov color palette

### PR DESCRIPTION
**Pull request by joshuami**

### CODE REVIEW

File: web/themes/custom/westy/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ignore SASS files
+/.sass-cache
+.DS_Store
+*.codekit
+.idea
+node_modules
+proxy.js
```
Comment (RichardDavies) [inline web/themes/custom/westy/.gitignore:7]: Needs a newline at EOF.

File: web/themes/custom/westy/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v14.15.4
```
Comment (RichardDavies) [inline web/themes/custom/westy/.nvmrc:1]: Needs a newline at EOF.

File: web/themes/custom/westy/assets/images/close-o.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    d="M16.3394 9.32245C16.7434 8.94589 16.7657 8.31312 16.3891 7.90911C16.0126 7.50509 15.3798 7.48283 14.9758 7.85938L12.0497 10.5866L9.32245 7.66048C8.94589 7.25647 8.31312 7.23421 7.90911 7.61076C7.50509 7.98731 7.48283 8.62008 7.85938 9.0241L10.5866 11.9502L7.66048 14.6775C7.25647 15.054 7.23421 15.6868 7.61076 16.0908C7.98731 16.4948 8.62008 16.5171 9.0241 16.1405L11.9502 13.4133L14.6775 16.3394C15.054 16.7434 15.6868 16.7657 16.0908 16.3891C16.4948 16.0126 16.5171 15.3798 16.1405 14.9758L13.4133 12.0497L16.3394 9.32245Z"
+    fill="currentColor"
+  />
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    d="M1 12C1 5.92487 5.92487 1 12 1C18.0751 1 23 5.92487 23 12C23 18.0751 18.0751 23 12 23C5.92487 23 1 18.0751 1 12ZM12 21C7.02944 21 3 16.9706 3 12C3 7.02944 7.02944 3 12 3C16.9706 3 21 7.02944 21 12C21 16.9706 16.9706 21 12 21Z"
+    fill="currentColor"
+  />
+</svg>
```
Comment (RichardDavies) [inline web/themes/custom/westy/assets/images/close-o.svg:18]: Needs a newline at EOF.

File: web/themes/custom/westy/assets/images/search-solid.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="search" class="svg-inline--fa fa-search fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"></path></svg>
```
Comment (RichardDavies) [inline web/themes/custom/westy/assets/images/search-solid.svg:1]: Needs a newline at EOF.

File: web/themes/custom/westy/templates/media/media--image--full.html.twig [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  set classes = [
+    'embed-image',
+    'embedded-full'
+  ]
+%}
+
+<div{{ attributes.addClass(classes) }}>
+  {{ title_suffix.contextual_links }}
+  {{ content }}
+</div>
```
Comment (RichardDavies) [inline web/themes/custom/westy/templates/media/media--image--full.html.twig:44]: Needs a newline at EOF.

**Review by RichardDavies — CHANGES_REQUESTED** [review]
I found a several files that need a newline. And it looks like no else has done any QA on this story yet--where you wanting me to do some as part of the PR review?

Also, I'm curious what your strategy and focus is for large PR's like this... since it's too big to give a detailed inspection of each file, what do you like to look for?

**Comment by joshuami** [comment]
> Also, I'm curious what your strategy and focus is for large PR's like this... since it's too big to give a detailed inspection of each file, what do you like to look for?

Mostly I'm looking for a sanity check. You can skip all the config files and really just take a high level look at the css mostly. If I didn't catch it in my extensive visual review of all the content types and views, then I'm not sure it is going to get caught at this point. That said, if you wanted to click through the multidev and just try to find things that you've worked on in the past that look broken, that would not be turned down.

Case in pointe, I swear I did a newline review and didn't catch a few I guess. Maybe I did the newline review on Kevin's story. It's been a big week. 😄 

I'll get you an updated PR in a couple of minutes with the newlines addressed.

**Comment by joshuami** [comment]
Newlines added. A couple of those files came with the theme, but correcting them is probably not a bad thing. 

If you are not seeing anything else that jumps out in a quick review, you are good to merge and deploy. If anything comes up, it will be on me to jump in and fix up.

**Review by RichardDavies — APPROVED** [review]

**Pull request merged**